website/docs/r/healthcare_fhir_service.html.markdown

```diff
@@ -0,0 +1,117 @@
+  resource_group_name = "tfex-resource_group"
+  workspace_id        = "tfex-workspace_id"
+  kind                = "fhir-R4"
+  authentication {
+    authority = "https://login.microsoftonline.com/tenantId"
+    audience  = "https://tfexfhir.fhir.azurehealthcareapis.com"
+  }
+  access_policy_object_ids = [
+    data.azurerm_client_config.current.object_id,
+  ]
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  acr_login_servers = ["tfex-container_registry_login_server"]
+
+  cors_configuration {
+    allowed_origins    = ["https://tfex.com:123", "https://tfex1.com:3389"]
+    allowed_headers    = ["*"]
+    allowed_methods    = ["GET", "DELETE", "PUT"]
+    max_age_in_seconds = 3600
+    allow_credentials  = true
+  }
+  export_storage_account_name = "storage_account_name"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Healthcare FHIR Service. Changing this forces a new Healthcare FHIR Service to be created.
+
+* `workspace_id`  - (Required) Specifies the name of the Healthcare Workspace where the Healthcare FHIR Service should exist. Changing this forces a new Healthcare FHIR Service to be created.
+
+* `location` - (Required) Specifies the Azure Region where the Healthcare FHIR Service should be created. Changing this forces a new Healthcare FHIR Service to be created.
+
+* `kind` - (Required) Specifies the kind of the Healthcare FHIR Service. Possible values are: `fhir-Stu3` and `fhir-R4`. Defaults to `fhir-R4`. Changing this forces a new Healthcare FHIR Service to be created.
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `access_policy_object_ids` - (Optional) A list of the access policies of the service instance.
+
+* `cors_configuration` - (Optional) A `cors_configuration` block as defined below.
+
+* `acr_login_servers` - - (Optional) A list of azure container registry settings used for convert data operation of the service instance.
+
+* `authentication` - (Required) An `authentication` block as defined below.
+
+* `export_storage_account_name` - (Optional) specifies the name of the export storage account which accepts the operation configuration information
+
+* `public_network_access_enabled` - (Optional) Whether to enabled public networks when data plane traffic coming from public networks while private endpoint is enabled.
+
+---
+An `identity` block supports the following:
+
+* `type` - (Required) The type of identity used for the Healthcare FHIR service. Possible values are `SystemAssigned`.
+
+---
+A `cors_configuration` block supports the following:  
+
+* `allowed_origins` - (Required) A set of origins to be allowed via CORS.
+* `allowed_headers` - (Required) A set of headers to be allowed via CORS.
+* `allowed_methods` - (Required) The methods to be allowed via CORS.
+* `max_age_in_seconds` - (Required) The max age to be allowed via CORS.
+* `allow_credentials` - (Boolean) If credentials are allowed via CORS.
```

Comment:
could we make this
```suggestion
* `credentials_allowed` - (Boolean) If credentials are allowed via CORS.
```

Reply:
also we do not add boolean here, this should be optional or required
```suggestion
* `allow_credentials` - (optional|required) If credentials are allowed via CORS.
```

Reply:
I used the same property definition from the existing resource [healthcare_service](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/healthcare_service), should I change that resource too?

Reply:
yes please